### PR TITLE
Implemented binary file sending functionality

### PR DESCRIPTION
**Pull request by oleksii-ponomarenko-checkr**
In the scope of working on my current task, I have faced a limitation that the existing attr `body_from_file` for HTTP protocol does not support binary files (PDF, DOC, ZIP, etc...) so I have expanded functionality with the new attribute `body_from_binary_file` which will send just binary file in HTTP protocol response.

### CODE REVIEW

File: load.go [hunk 1]
```diff
@@ -264,6 +272,21 @@ func readFile(templateKey string, baseDir string, filePath string) string {
 
 }
 
+func readBinaryFile(templateKey string, baseDir string, filePath string) []byte {
```
Comment (jpatdalton) [inline load.go:275]: It feels like the function above this is doing the same thing, but simply returning to a string - I'm wondering if this warrants a completely new function or whether there can be an argument for parsing the body from a file with the return data type (and it defaults to a string).

**Review by sesquipedalian-dev — DISMISSED** [review]

File: load.go (hunk 1 above)
Comment (oleksii-ponomarenko-checkr) [reply load.go:275]: Ok, I will try to modify the existing readFile function.

Comment (oleksii-ponomarenko-checkr) [reply load.go]: @jpatdalton Done, you can look at new changes.

**Comment by oriahu** [comment]
LGTM

**Review by oriahu — APPROVED** [review]

**Pull request merged**